plugins/inventory/proxmox.py

```diff
@@ -323,8 +323,10 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
 
                 # Additional field containing parsed tags as list
                 if config == 'tags':
-                    parsed_key = self.to_safe('%s%s' % (key, "_parsed"))
-                    properties[parsed_key] = [tag.strip() for tag in value.split(",")]
+                    stripped_value = value.strip()
+                    if stripped_value:
+                        parsed_key = key + "_parsed"
+                        properties[parsed_key] = [tag.strip() for tag in stripped_value.split(",")]
```

Comment:
Why not
```suggestion
                    parsed_key = key + "_parsed"
                    properties[parsed_key] = [tag.strip() for tag in stripped_value.split(",")] if stripped_value else []
```
?

Reply:
Because I find that form harder to read. Up to you, I'll commit whichever you prefer.

Reply:
Well, your code omits `properties[parsed_key]` when there is no tag. I would say that's wrong.

Reply:
That's the behaviour of the current version, though. When there are no tags, there's no `proxmox_tags_parsed` output.

Reply:
Ah, that's helpful. That wasn't exactly clear from the code ;-) I'd still say that it *should* always return `proxmox_tags_parsed` (as an empty list if there are no tags), but that would be a new feature then.